bridge/cypress/integration/page-data-update-project-switch.spec.ts

```diff
@@ -0,0 +1,88 @@
+/// <reference types="cypress" />
+import EnvironmentPage from 'cypress/support/pageobjects/EnvironmentPage';
+import ServicesPage from 'cypress/support/pageobjects/ServicesPage';
+import SettingsPage from 'cypress/support/pageobjects/SettingsPage';
+import UnifromPage from 'cypress/support/pageobjects/UniformPage';
+import BasePage from '../support/pageobjects/BasePage';
+
+describe('verify Page data update when project is switched', () => {
+  it('testing the data change/update ', () => {
+    const basePage = new BasePage();
+    const envPage = new EnvironmentPage();
+    const servicePage = new ServicesPage();
+    const uniformPage = new UnifromPage();
+    const settingsPage = new SettingsPage();
+    const PROJECT_DYNATRACE = 'dynatrace';
+    const PROJECT_TEST = 'testproject';
+    const QG_DYNATRACE = 'quality-gate';
+    const QG_TEST_PROJ = 'newproj-quality-gate';
+    const SERVICE_DYNATRACE = 'items';
+    const SERVICE_TEST_PROJ = 'newService';
+    const UNIFORM_REMEDIATION = 'remediation-service';
+    const UNIFORM_APPROVAL_SERVICE = 'approval-service';
+    const UNIOFORM_DYNATRACE_SERVICE = 'dynatrace-service';
```

Comment:
Typo ...
`UNIOFORM_DYNATRACE_SERVICE` > `UNIFORM_DYNATRACE_SERVICE`

Reply:
Thanks for the finding, fixed!